QT/wavedisplay.cpp

```diff
@@ -109,3 +134,12 @@ void WaveDisplay::onPlayButtonClicked()
 {
     playSound();
 }
+
+void WaveDisplay::onSpinBoxChanged(double value)
```

Comment:
I think we need to limit the rate at which things get updated to prevent the program from crashing. I'll look into it